Dockerfile-cli

```diff
@@ -4,10 +4,10 @@ FROM php:7.2-cli-alpine3.7 as cli
 # Add usabilla user and group
 RUN set -x \
     && addgroup -g 1000 app \
-    && adduser -u 1000 -D -G app app
+    && adduser -u 1000 -D -G app app \
 
-# Temporary fix: pulls in the aports patch for https://bugs.alpinelinux.org/issues/10648
-RUN apk add --no-cache --upgrade apk-tools
+    # Temporary fix: pulls in the aports patch for https://bugs.alpinelinux.org/issues/10648
```

Comment:
Not directly related to this PR, but I was curious to see whether we still need this fix. The link reference is broken, but I think found the [new location](https://gitlab.alpinelinux.org/alpine/apk-tools/-/issues/10648) of this issue. Looks like we might not need to pull in `apk-tools` any more. Shall we test this?